doc/cascadia/Cascading-Default-Settings.md

```diff
@@ -0,0 +1,571 @@
+`Program Files(x86)`.
+
+Additionally, if it _is_ installed, we set it as the default profile instead of
+Windows Powershell.
+
+Should we include it in the default profiles? If we wanted to do this, we'd need
+to:
+* hide it in the user settings if it's not installed.
+* Set the commandline in the user settings. If the user deletes this value, then
+  that profile would be unable to launch, unlike all the other profiles that are
+  in the user settings (by default).
+* Make sure to set the default profile in the user settings.
+
+Or, should we only add it to the user settings?
+* We'd only generate it in the user settings if it's installed.
+* Set the commandline in the user settings. If the user deletes this value, then
+  that profile would be unable to launch, unlike all the other profiles that are
+  in the user settings (by default).
+* Make sure to set the default profile in the user settings.
+
+Currently there are only a few things that we actually set for the Powershell
+Core profile - the name, commandline, icon, and the color scheme. I don't
+believe it would add too much bloat to the user's settings to generate it in the
+user's settings by default.
+
+As a third option, we could treat the Powershell Core profile like a dynamic
+profile, and have it autogenerated if it exists, and auto-hidden if it's
+uninstalled. We'd need to additionally include `autoloadPowershellCoreProfile`,
+set to true in the default settings. However, none of the other profile
+generators need to be able to change the default profile.
+
+Considering that the first option would still need to create an entry for
+powershell core in the user's settings file, I'd lean towards the third option -
+making it a dynamic profile just like WSL distros.
+
+### Unbinding a Keybinding
+
+How can a user unbind a key that's part of the default keybindings? What if a
+user really wants <kbd>ctrl</kbd>+<kbd>t</kbd> to fall through to the
+commandline application attached to the shell, instead of opening a new tab?
+
+We'll need to introduce a new keybinding command that should indicate that the
+key is unbound. We'll load the user keybindings and layer them on the defaults
+as described above. If during the deserializing we find an entry that's bound to
+the command `"unbound"` or any other string that we don't understand, instead of
+trying to _set_ the keybinding, we'll _clear_ the keybinding with a new method
+`AppKeyBindings::ClearKeyBinding(chord)`.
+
+### Removing the Globals Object
+
+As a part of #[1005](https://github.com/microsoft/terminal/pull/1005), all the
+global settings were moved to their own object within the serialized settings.
+This was to try and make the file easier to parse as a user, considering global
+settings would be intermingled with profiles, keybindings, color schemes, etc.
+Since this change will make the user settings dramatically easier to navigate,
+we should probably remove the `globals` object, and have globals at the root
+level again.

```

Comment:
this feels optional to me.

Reply:
Yes this is a totally optional change. Kinda threw it out there to see how people felt. Thoughts?

Reply:
Eh, I think you should leave it until proven otherwise.